Scripts/ci.sh

```diff
@@ -1,8 +1,18 @@
 #!/bin/bash -l
 set -ex
 
-if [ -n "$DESTINATION" ]; then
-  xcodebuild -project Valet.xcodeproj -scheme "$SCHEME" -sdk $SDK -destination "$DESTINATION" -configuration Debug -PBXBuildsContinueAfterErrors=0 $ACTION
-else
-  xcodebuild -project Valet.xcodeproj -scheme "$SCHEME" -sdk $SDK -configuration Debug -PBXBuildsContinueAfterErrors=0 $ACTION
+if [ $ACTION == "xcode" ]; then
+  if [ -n "$DESTINATION" ]; then
+    xcodebuild -UseModernBuildSystem=NO -project Valet.xcodeproj -scheme "$SCHEME" -sdk $SDK -destination "$DESTINATION" -configuration Debug -PBXBuildsContinueAfterErrors=0 $XCODE_ACTION
+  else
+    xcodebuild -UseModernBuildSystem=NO -project Valet.xcodeproj -scheme "$SCHEME" -sdk $SDK -configuration Debug -PBXBuildsContinueAfterErrors=0 $XCODE_ACTION
```

Comment:
Note that we need to skip using the modern build system because the modern build system builds our custom `XCTest` library for non-watch targets for some unknown reason. Once the `XCTest` target is in derived data, the new build system uses it instead of the system's `XCTest`. Using the old build system fixes the issue.